README.md

```diff
@@ -194,6 +194,11 @@ If the `required` prop is set to true, the input will have to be filled before t
   required: PropTypes.bool,
 ```
 
+The `screenReaderInputMessage` prop allows you to write a message for screen readers. When the prop is added a `aria-describedby` attribute will be added to the input box and the value will be read by screen readers. Take advantage of this to inform users about constraints your component has, such as the date format, minimum nights and blocked out dates etc.
```

Comment:
Would like to see a few grammatical changes here, but overall looks like reasonable documentation! Thanks for including this. Maybe something a bit more in this direction?

> The `screenReaderInputMessage` prop accepts a contextual message for screen readers. When an input is focused, the `screenReaderInputMessage` prop value is read. This can inform users about constraints, such as the date format, minimum nights, blocked out dates, etc.